diff --git a/dist/providers/command-runner.service.mjs b/dist/providers/command-runner.service.mjs
index v1.0.0-alpha.1..v1.0.0-alpha.1 100644
--- a/dist/providers/command-runner.service.mjs
+++ b/dist/providers/command-runner.service.mjs
@@ -15,1 +15,1 @@
-import { cast, constant, define, entries, fallback, flat, get, isString, isUndefined, lowercase, pick, select } from "@flex-development/tutils";
+import { cast, constant, define, entries, fallback, flat, get, isString, isUndefined, lowercase, noop, pick, select } from "@flex-development/tutils";
@@ -76,1 +76,8 @@
-      env && option.env(env), choices.length && option.choices(choices), option.argParser(method.handler.bind(metadata.instance)), command.addOption(option);
+      env && option.env(env);
+      choices.length && option.choices(choices);
+      const { parseArg = noop } = option;
+      option.argParser((value, previous) => {
+        parseArg(value, previous)
+        return method.handler.bind(metadata.instance)(value, previous)
+      });
+      command.addOption(option);
